docker/install/ubuntu_install_nnpack.sh

```diff
@@ -26,7 +26,7 @@ git clone https://github.com/Maratyszcza/NNPACK NNPACK
 git clone https://github.com/Maratyszcza/pthreadpool  NNPACK/pthreadpool
 
 # Use specific versioning tag.
-(cd NNPACK && sed -i 's/GIT_TAG master/GIT_TAG main/g' ./cmake/DownloadCpuinfo.cmake && git checkout 1e005b0c2)
+(cd NNPACK && git checkout 1e005b0c2)
```

Comment:
If the old SHA is used, the old branch of `CPUInfo` will get picked again. Old problem of `nnpack` build failure in TVM will reappear. Shouldn't we use the latest nnpack SHA i.e., `70a77f485e8b934224f3a79efd8edcd84cd377b8` that reflects the change to pick up thew new branch name of `CPUInfo`?

Reply:
You are right. Thanks @ashutosh-arm , I have fixed the patch with the latest commit.